src/Sarif.UnitTests/Writers/SarifLoggerTests.cs

```diff
@@ -186,6 +186,51 @@ public void SarifLogger_WritesFileData()
             sarifLog.Runs[0].Files[fileDataKey].Hashes[2].Value.Should().Be("0953D7B3ADA7FED683680D2107EE517A9DBEC2D0AF7594A91F058D104B7A2AEB");
         }
 
+        [Fact]
+        public void SarifLogger_WritesFileDataWithUnrecognizedEncoding()
+        {
+            var sb = new StringBuilder();
+            string file;
```

Comment:
Declare at point of use.

Reply:
It's used outside the using block, down on line 225.

Reply:
I missed that.